docs/navigate.rst

```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+              ]
+          },
+          "My_System": {
+              "class": "Telemetry_System",
+              "enable": "true",
+              "systemPoller": ["My_Poller"]
+          },
+          "My_Pull_Consumer": {
+              "class": "Telemetry_Pull_Consumer",
+              "type": "default",
+              "systemPoller": ["My_Poller"]
+          }
+      }
+
+5. If further customization is required to filter only specific virtual servers and pools, refer to this link: https://clouddocs.f5.com/products/extensions/f5-telemetry-streaming/latest/data-modification.html
+ 
+View VIP Dashboard
+-------------------
+
+1. Click the Visibility tab, and then from the Table list, click the VIP table.
+
+2. From the Partition list, click the partition of interest.
+
+3. The VIP table for this BIG-IP and partition will be displayed.
+
+4. Click the VIP for which the dashboard is to be displayed. This redirects to the Visibility Dashboard sub-tab. 
+   
+   .. note::
+   
+      - It is possible to directly click the **Visibility Dashboard** sub-tab and then select the VIP, instead of a redirect from **Visibility Table**.
+      
+
+5. Visibility Dashboard displays the information for the selected VIP including VIP name, service port, protocol, SNAT, route domain, iRules, default pool (and pool information such as load balancing and pool monitor)
+         
+         a. Dashboard displays a **Telemetry Consumer** drop-down list. Select the appropriate consumer from which the statistics need to be obtained. The Virtual pool statistics will be displayed on the dashboard including Bits, Packets, Connections and Requests. 
+         
+            .. note::
+         
+               - Telemetry plugin needs to be installed on the BIG-IP to be able to view the Telemetry consumer list and stats from the telemetry consumer. Check the **Configure Telemetry** section for installation and configuration.
+            
+               - If there is only a single Telemetry pull consumer configured, then FASC will directly query that consumer and display the stats.
+            
+         b. **View Stats** - Dashboard displays a **View Stats** link besides the default pool. It displays total pool member count and the availability and enabled state of the pool  members using a pie chart.
+         
+         c. **View Logs** - Dashboard displays a **View Logs** link besides the VIP, which, on click will open a popup window and display all the logs for the selected Virtual Server. 
+         
+            .. note::
+
+               - It is possible to also view all logs instead of the filtered logs.
+            
+               - It is possible to specify a UTC from and to Date/Time for the logs to be retrieved. The From/To time should be specified in UTC timezone. The response too will be in UTC.
+
+         d. **View Connections** - Dashboard will display a **View Connections** link which, on click will display all the active connections to that VIP.
```

Comment:
The Dashboard displays a **View Connections** link, which, when clicked displays all the active...